core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -355,7 +355,7 @@ object TopicCommand extends Logging {
     override def describeTopic(opts: TopicCommandOptions): Unit = {
       val topics = getTopics(opts.topic, opts.excludeInternalTopics)
       val topicOptWithExits = opts.topic.isDefined && opts.ifExists
-      ensureTopicExists(topics, topicOptWithExits)
+      ensureTopicExists(topics, opts.topic, topicOptWithExits)
```

Comment:
typo: should be `topicOptWithExists`

Reply:
Yeah you are right. I did not rename the variable since it was already there. Let me do that.

Reply:
Used `requireTopicExists` rather than `topicOptWithExists`. Felt like the previous one was not a very descriptive name.